suite-native/app/android/app/src/main/res/xml/device_filter.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <usb-device vendor-id="21324" product-id="1" />
+    <usb-device vendor-id="4617" product-id="21440" />
+    <usb-device vendor-id="4617" product-id="21441" />
```

Comment:
Equivalent of our 3 devices?

Reply:
Good catch, TS3 is missing here, first two are T1 I think. I will verify this once more.